tests/pytorch/fpga/test_reshape_fpga.py

```diff
@@ -0,0 +1,125 @@
+# Simple test for relu for FPGA
+
+# TODO: conform to pytest syntax if needed
+
+from dace.transformation.interstate import FPGATransformSDFG, InlineSDFG
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import onnx
+import numpy as np
+
+import daceml.onnx as donnx
+from daceml.pytorch import DaceModule, dace_module
+from daceml.onnx import ONNXModel
+import copy
+import dace
+import argparse
+import onnx
+from daceml.util import utils
+from multiprocessing import Process, Queue
+
+
+class Model(nn.Module):
+    def __init__(self, new_shape):
+        super(Model, self).__init__()
+        self.new_shape = new_shape
+
+    def forward(self, x):
+        x = x.reshape(self.new_shape)
+        return x
+
+
+def run(data_shape: tuple, reshaped_shape: tuple, vec_width=1, queue=None):
+    # dace_output = dace_model(x)
+
+    import daceml.onnx as donnx
+    donnx.default_implementation = "pure"
+    ptmodel = Model(reshaped_shape)
+    x = torch.rand(data_shape)
+
+    torch_output = ptmodel(x)
+
+    dace_model = DaceModule(ptmodel)
+    out = dace_model(x)
+    sdfg = dace_model.sdfg
+    sdfg.apply_transformations([FPGATransformSDFG])
+
+    donnx.ONNXReshape.default_implementation = 'fpga'
+    sdfg.expand_library_nodes()
+    sdfg.apply_transformations_repeated([InlineSDFG])
+
+    dace_output_fpga = dace_model(x)
+    dace_output_fpga = dace_output_fpga.reshape(
+        torch_output.detach().numpy().shape)
```

Comment:
```suggestion
        torch_output.detach().numpy().shape).detach().numpy()
```
Need to detach and convert the reshaped tensor to NumPy, to be able to use `np.linalg.norm` on the result below.

Reply:
Thanks!